src/zhinst/toolkit/helpers/sequences.py

```diff
@@ -1055,53 +1056,103 @@ def update_params(self):
 
 @attr.s
 class CustomSequence(Sequence):
-    """A *Custom Sequence* for compiling an existing `.seqC` program.
+    """A *Custom Sequence* for compiling an existing AWG program.
 
-    The *Custom Sequence* allows the user to specify the file path to an
-    existing `.seqC` program. It needs to be located in the folder
-    *".../Zurich Instruments/LabOne/WebServer/awg/src"*.
+    The *Custom Sequence* allows the user to specify the Python string
+    variable that stores the AWG program or a path to an existing
+    `.seqC` file.
 
-    Additionally, the *Custom Sequence* gives the user the ability to define
-    variable placeholders in their `.seqC` program. So parameter `custom_params`
-    expects a list of values that replace placeholders in the program. The
-    placeholders are specified as special in the format `"$param{i}$"` where `i`
-    is the index of the value in the *custom_params* list.
+    Additionally, the *Custom Sequence* gives the user the ability to
```

Comment:
can you use another word instead of placeholder? To avoid confusion with the keyword placeholder to deplacre waveforms